modules/form-fields/package.json

```diff
@@ -0,0 +1,45 @@
+{
+    "name": "@wso2is/form-fields",
+    "version": "1.0.0",
+    "description": "Form library for WSO2 Identity Server front end applications.",
+    "keywords": [
+        "WSO2",
+        "WSO2 IS",
+        "WSO2 Identity Server",
+        "WSO2 Identity Apps",
+        "form"
+    ],
+    "main": "dist/src/index.js",
+    "types": "dist/typings/index.d.ts",
+    "scripts": {
+        "prebuild": "npm run clean",
+        "build": "npm run compile",
+        "clean": "rimraf ./dist",
+        "compile": "npx run-script-os",
+        "compile:win32": "..\\..\\node_modules\\.bin\\tsc -p tsconfig.json",
+        "compile:default": "../../node_modules/.bin/tsc -p tsconfig.json",
+        "lint": "tslint -t stylish --project .",
+        "test": "npm run jest --passWithNoTests",
+        "jest": "npx run-script-os",
+        "jest:win32": ".\\node_modules\\.bin\\jest",
+        "jest:default": "./node_modules/.bin/jest"
+    },
+    "dependencies": {
+        "@types/jest": "^26.0.14",
+        "@wso2is/core": "^1.2.107",
+        "@wso2is/react-components": "^1.2.107",
+        "@wso2is/validation": "^1.2.107",
+        "jest": "^26.4.2",
+        "joi": "^17.4.0",
+        "ts-jest": "^26.4.0"
+    },
+    "devDependencies": {
+        "rimraf": "^3.0.2",
+        "run-script-os": "^1.0.7"
+    },
+    "peerDependencies": {
+        "classnames": "*"
```

Comment:
Lets move `classnames` to dependencies and use the exact version used in parent package.json so that hoisting works as expected.

And also add `react` as a `peerDependencies`.